TLM/TMPE.CitiesGameBridge/TMPE.CitiesGameBridge.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What exactly changed here, is it BOM?

Reply:
Yeah. I tried to add a new file but then changed my mind and deleted it. that's why the bom has changed.